src/platform/utilities/api/index.js

```diff
@@ -23,6 +23,7 @@ function isJson(response) {
 export function apiRequest(resource, optionalSettings = {}, success, error) {
   const baseUrl = `${environment.API_URL}/v0`;
   const url = resource[0] === '/' ? [baseUrl, resource].join('') : resource;
+  const isLogout = resource.indexOf('/slo/') !== -1;
```

Comment:
We might want to be more specific here. For now, we only have `/sessions/slo/new`, but if any future endpoint has the string 'slo', this could break.